docs/release-notes.mdx

```diff
@@ -167,9 +166,9 @@ mode.
 
 Adds support for Linea ENS domains, including a frontend app for users to register and manage their
 domains. The Linea ENS system is comparable to ENS on Ethereum Mainnet, except that domains are 
-limited to one per account and registering requires completion of [Proof of Humanity](https://poh.linea.build/).
-As the system leverages CCIP Read, which enables L1 to trustlessly query the Linea ENS registry 
-through an offchain gateway, Linea ENS domains resolve on L1. 
+limited to one per account, and registering requires completion of [Proof of Humanity](https://poh.linea.build/).
+As the system leverages CCIP Read, which enables L1 to seamlessly query the Linea ENS registry 
```

Comment:
```suggestion
As the system leverages CCIP Read, which enables L1 to trustlessly query the Linea ENS registry 
```

This would completely change the meaning -- `trustless` is also intended here